winpr/include/winpr/stream.h

```diff
@@ -371,18 +371,24 @@ extern "C"
 		_s->length = (size_t)(_s->pointer - _s->buffer);
 	}
 
-	static INLINE size_t Stream_GetRemainingCapacity(wStream* _s)
+	static INLINE ssize_t Stream_GetRemainingCapacity(wStream* _s)
```

Comment:
`ssize_t` is not avialable on older visual studio compilers
`ssize_t` only defines `-1` as negative value (others might not be there)

Reply:
OK, understood.